.pre-commit-config.yaml

```diff
@@ -59,7 +59,7 @@ repos:
       language_version: python3
       types: [python]
 - repo: https://github.com/Lucas-C/pre-commit-hooks-safety
-  rev: v1.2.1
```

Comment:
@nocturn9x ok. Si applica anche a https://github.com/par-tec/python-cookiecutter ?

Reply:
Immagino di sì: il problema è dovuto a un cambiamento del pyproject.toml di poetry a partire da una specifica versione, quindi tutto ciò che dipende da poetry in generale si rompe

Reply:
@nocturn9x poetry not for prod per ora ;)